### PR TITLE
updated fastify version

### PR DESCRIPTION
**Pull request by imrefazekas**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,6 +18,6 @@ module.exports = fp((fastify, opts, next) => {
 
   next()
 }, {
-  fastify: '1.x',
+  fastify: '>=1.x',
```
Comment (gj) [inline index.js:21]: I think I'd feel better about not making this quite so open-ended. What are your thoughts?

File: package.json [hunk 2]
```diff
@@ -26,18 +26,18 @@
   },
   "homepage": "https://github.com/gj/fastify-ws#readme",
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "fastify": "^1.7.0",
+    "eslint": "^5.14.1",
+    "fastify": "^2.0.0",
```
Comment (gj) [inline package.json:30]: I wonder if we should change this to something like `"fastify": "1.7.0 - 2"` to bring these package bumps to users of Fastify 1.x as well. If there's no breaking change with the Fastify API that we're interfacing with (need to test this out later), I think I'd prefer to maintain compatibility.

**Review by gj — COMMENTED** [review]
Hey @imrefazekas — thanks for the contribution!

I'll test this out after work today.

**Comment by imrefazekas** [comment]
Pls merge and publish if it works fine. It is working in my app using my branch...
Thanks!

File: index.js (hunk 1 above)
Comment (gj) [inline index.js:21]:
```suggestion
  fastify: '1.7.0 - 2',
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-ws",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (gj) [inline package.json:3]: I'll bump the package version in a separate commit
```suggestion
  "version": "1.0.0",
```

File: package.json (hunk 2 above)
Comment (gj) [inline package.json:30]:
```suggestion
    "fastify": "1.7.0 - 2",
```

**Review by gj — CHANGES_REQUESTED** [review]
Would like to nail down the version compat a bit tighter, but once these changes are in I'll merge and publish. Thanks again for the contribution @imrefazekas !

**Pull request merged**